src/matrix.ts

```diff
@@ -108,7 +107,7 @@ export function unset<T>(
 }
 
 export function reduce<T, A>(
-  func: (A, T | typeof undefined, Types.Point) => A,
+  func: (_: A, __: T | typeof undefined, arg2: Types.IPoint) => A,
```

Comment:
Name these so that usage hints at least have helpful docs.
```suggestion
  func: (acc: A, value: T | typeof undefined, point: Types.IPoint) => A,
```